lib/libutee/tee_api_operations.c

```diff
@@ -1100,13 +1087,15 @@ TEE_Result TEE_CipherDoFinal(TEE_OperationHandle operation,
 	} else {
 		req_dlen = srcLen;
 	}
-	if (*destLen < req_dlen) {
-		*destLen = req_dlen;
+	if (destLen)
```

Comment:
It looks like this is a bug fix, should the case `destLen == NULL` be handled in a previous commit instead?

Reply:
`destLen` is not feed directly to the syscall, it's done via a temporary variable so no other direct checks are done on this address.

Reply:
Right, that's fine then.